components/eamxx/src/share/io/scream_output_manager.cpp

```diff
@@ -560,7 +560,7 @@ void set_file_header(const std::string& filename)
   set_str_attribute_c2f(filename.c_str(),"institution_id","");  // TODO
   set_str_attribute_c2f(filename.c_str(),"product","");  // TODO
   set_str_attribute_c2f(filename.c_str(),"component","ATM");
-  set_str_attribute_c2f(filename.c_str(),"conventions","");  // TODO
+  set_str_attribute_c2f(filename.c_str(),"conventions","CF-1.0");  // TODO: It might be nice to have this passed from CIME.  We hard-code for now though, because post-processing needs something in this global attribute. 2023-04-12
```

Comment:
One comment on a one-line PR: I disagree with the comment that CIME should tell us the CF convention value... that value is determined by what output units, what time bounds, and what other metadata we have. That's EAMxx's to determine, not CIME.

Reply:
Agreed. CIME probably doesn't even know what CF is, not should it.

Reply:
>One comment on a one-line PR:

It's okay, every revision arbitrarily buffs my git stats with an extra commit ;-)